src/Auth0Client.ts

```diff
@@ -939,6 +950,15 @@ export default class Auth0Client {
       if (e.message === MISSING_REFRESH_TOKEN_ERROR_MESSAGE) {
         return await this._getTokenFromIFrame(options);
       }
+      // A refresh token was found, but is it no longer valid.
+      // Fallback to an iframe.
+      if (
+        e.message &&
+        e.message.indexOf(INVALID_REFRESH_TOKEN_ERROR_MESSAGE) > -1
+      ) {
```

Comment:
Maybe we can unify this `if` block with the conditional above it, rather than having two if's? Feels like we could do it with an or expression.